sympy/series/tests/test_gruntz.py

```diff
@@ -467,3 +467,7 @@ def test_issue_6682():
 def test_issue_7096():
     from sympy.functions import sign
     assert gruntz(x**-pi, x, 0, dir='-') == oo*sign((-1)**(-pi))
+
+def test_issue_18482():
+    _p = Symbol('_p', positive=True)
```

Comment:
This needs to be removed

Reply:
Shall I use x instead of _p?

Reply:
You have defined  `_p` but used x. Anyway, use `x` and remove this line